cv32e40s/tests/programs/custom/custom_interrupt_test/README.md

```diff
@@ -0,0 +1,3 @@
+CV32E40S Core Interrupt Testcases
+=============================
+This directory contains manually written directed testcases for interrupt testing of CV32E40S.
```

Comment:
Could you add some more info in the readme, about why this test exists and approximately what it does?
(I know that is more documentation than some of the other tests have, but I think it would be useful for a future person that stumbles upon this code.)

Reply:
This directed test was only used while debugging to test for some specific cases. I am not sure if it really is necessary, so I am considering removing it

Reply:
I lean towards adding the new test, because the cost of adding it is relatively low and it provides additional test stimuli.